yt_dlp/extractor/aiyifan.py

```diff
@@ -0,0 +1,109 @@
+import hashlib
+import re
+import time
+
+from yt_dlp.extractor.common import InfoExtractor
+
+
+class AiyifanIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?yfsp\.tv/play/(?P<id>[^/?#]+)(?:\?id=(?P<alt_id>[^&#]+))?'
+    # All the three domains are the same
```

Comment:
This comment is unclear to me. I only see one domain in the `_VALID_URL` and `_TESTS`